static/slides/intro_reproducibility_Rmd/intro_reproducibility.Rmd

```diff
@@ -134,41 +134,17 @@ the computer to do</font>.* </center> (Donald E. Knuth, Literate Programming,
 
 ## Literate programming in <img src="../../img/slides/Rlogo.png" width="50px">
 
-> - **Sweave** (`.Rnw`): LaTeX + <img src="../../img/slides/Rlogo.png" width="40px">
-<br><br>Example:<br>
-`knitr::knit2pdf("foo.Rnw")` $\rightarrow$ `foo.pdf`<br>
-
 <br>
 
 > - **Rmarkdown** (`.Rmd`): markdown + <img src="../../img/slides/Rlogo.png" width="40px">
 <br><br>Example:<br>
-`knitr::knit2html("foo.Rmd")`  $\rightarrow$  `foo.html`<br>
+`rmarkdown::render("foo.Rmd")`  $\rightarrow$  `foo.html`<br>
 `rmarkdown::render("foo.Rmd")`  $\rightarrow$  `foo.pdf`<br>
 `rmarkdown::render("foo.Rmd")`  $\rightarrow$  `foo.doc`<br>
 `...`
 
 
 
-
-## **Sweave**: <img src="../../img/slides/Rlogo.png" width="50px"> chunks in LaTeX {.smaller}
```

Comment:
Some people still use LaTeX, in which case `knit2pdf` remains a good option for generating documents. I think I'd rather leave it in for now

Reply:
Perhaps adding it in as a footnote is a good option since the audience for this tutorial is largely non-technical?

Reply:
Could do. Do you know on the top of your head how to do it? Or feel free to PR ;)

Reply:
Usually something like this [^1]

[^1]: this referring to a footnote, not the actual word [this]

[this]: https://pandoc.org/MANUAL.html#footnotes